Makefile

```
@@ -0,0 +1,28 @@
+.SILENT:
+.PHONY: compile
+SOURCE_FOLDER=./src/openzeppelin
+
+install:
+	git submodule init && git submodule update && cp -rf cairo/corelib .
+
+update:
+	git submodule update && cp -rf cairo/corelib .
```

Why no do 
```suggestion
install: init update
init:
	git submodule init
update:
	git submodule update
	cp -rf cairo/corelib .
```
and avoid code dupliation